.github/workflows/codespell.yml

```diff
@@ -0,0 +1,27 @@
+# GitHub Action to automate the identification of common misspellings in text files.
+# https://github.com/codespell-project/actions-codespell
+# https://github.com/codespell-project/codespell
+name: codespell
+on:
+  push:
+    branches:
+      - main
+      - '[0-9]+.x'
```

Comment:
Not sure this is correct. Should at least contain master but [0-9]+.x I don't think applies to our namings

Reply:
yeah, this is fixed now.